### DISCUSSION TITLE
Expand ARS area

### DISCUSSION THREAD

**Discussion by Fristorm-Palifen** [eccentricdevotion/TARDIS#414] [Ideas]
Why not expand the ARS area for tardis in custom world ? because no other tardis can grow on it! and with Massive console we can't create many room xD

**Comment by eccentricdevotion** [comment]
Because you're not limited to growing rooms to just inside the ARS grid, you can manually grow rooms outside it.

**Reply by Fristorm-Palifen** [reply]
ohh ok
